documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -54,15 +54,23 @@ You have to first downgrade through one of the Strimzi versions in-between, disa
 The `UseKRaft` feature gate has a default state of _disabled_.
 
 The `UseKRaft` feature gate deploys the Kafka cluster in the KRaft (Kafka Raft metadata) mode without ZooKeeper.
+ZooKeeper and KRaft are mechanisms used to manage metadata and coordinate operations in Kafka clusters. 
+KRaft mode eliminates the need for an external coordination service like ZooKeeper. 
+In KRaft mode, brokers take on the roles of brokers, controllers, or both. 
+They collectively manage the metadata, which is replicated across partitions. 
+Controller brokers are responsible for coordinating operations and maintaining the cluster's state.
+
 This feature gate is currently intended only for development and testing.
 
-IMPORTANT: **The KRaft mode is not ready for production in Apache Kafka or in Strimzi.**
+IMPORTANT: **KRaft mode is not ready for production in Apache Kafka or in Strimzi.**
 
 When the `UseKRaft` feature gate is enabled, the Kafka cluster is deployed without ZooKeeper.
-*The `.spec.zookeeper` properties in the Kafka custom resource will be ignored, but still need to be present.*
+*The `.spec.zookeeper` properties in the `Kafka` custom resource are ignored, but still need to be present.*
 The `UseKRaft` feature gate provides an API that configures Kafka cluster nodes and their roles.
 The API is still in development and is expected to change before the KRaft mode is production-ready.
 
+If you are trying Kafka in KRaft mode, you can also enable the `KafkaNodePools` feature gate to manage Kafka node configurations in node pools. 
+
 Currently, the KRaft mode in Strimzi has the following major limitations:
 
 * Moving from Kafka clusters with ZooKeeper to KRaft clusters or the other way around is not supported.
```

Comment:
We should probably add to the limitations that `controller` only nodes cannot be rolled or updated.